src/headers.toit

```diff
@@ -5,17 +5,30 @@
 import bytes
 
 class Headers:
-  headers_ := Map
+  headers_/Map? := null
 
   constructor:
 
+  constructor.from_map map/Map:
+    map.do: | key value |
+      if key is not string:
```

Comment:
Would be better to type key as a string, I think.

Reply:
done.

Reply:
also added an `else` to the value type checks to throw when the value is neither a string nor a list.